### ISSUE TITLE
Mismatch between coach Reports and generated CSV

### ISSUE THREAD

**Issue by jonboiser** [learningequality/kolibri#6642]
### Observed behavior

1. On ReportsLessonExerciseLearnerListPage, if "view by groups" checkbox is toggled, ungrouped learners aren't in the report. The report should be the same regardless of view (ungrouped learners appear in the report even if "view by groups is checked")

Screenshot
![Screen Shot 2020-03-09 at 4 05 45 PM](https://user-images.githubusercontent.com/10248067/76264826-17715400-6220-11ea-83f7-f2937763f419.png)

Exported CSV

```
"Group name","Name","Progress","Time spent","Last activity"
```

~2. ReportsQuizLearnerListPage - there is a subtile under "progress" saying e.g. "3 of 10 questions answered", which isn't on the CSV. Might be useful.~ Added some extra columns in this report (total/answered/correct questions)

<img width="947" alt="Screen Shot 2020-03-12 at 1 20 38 PM" src="https://user-images.githubusercontent.com/10248067/76563341-6f989800-6464-11ea-8749-37cd7f033f48.png">

3. QuizReportsListPage (and others) - There's no "Total learners" field, just tallies for different statuses, whereas "Difficult Questions" on exam report does show tally for all learners.

4. Some numbers and percentages are being printed as strings e.g. `"50%"` and `"10"` instead of as numbers. Might not be auto-parsed correctly in some software.

### Expected behavior

…

### User-facing consequences

…

### Errors and logs

…

### Steps to reproduce

…

### Context

…

**Comment by rtibbles** [comment]
@radinamatic @pcenov when reviewing this issue, could you do so on the develop branch? The latest 0.16 alpha asset would suffice.

**Comment by pcenov** [comment]
@rtibbles retested in the latest [0.16 build](https://buildkite.com/learningequality/kolibri-python-package/builds/5161) and the issues listed above are still valid.

Here's a screenshot for point 1:
![2022-03-24_13-51-29](https://user-images.githubusercontent.com/79847249/159911715-d9fcda5a-7a4d-44ae-a9d6-5813a4db50e3.png)
Here it should be noted also that the 'Last activity' column is empty.

The option to export a quiz report as CSV is not functioning at all:
![2022-03-24_13-55-36](https://user-images.githubusercontent.com/79847249/159911577-d0ed8f58-256b-49fb-8fec-8b34814be91f.png)

Average score value not displayed correctly:
![2022-03-24_14-08-33](https://user-images.githubusercontent.com/79847249/159913416-a71872a8-aa5d-492a-b42a-3ac5948c9f41.png)

**Comment by AlexVelezLl** [comment]
Closed by #12628
